# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/test/crashtests/crashtests.list
# Commit: c11a6ced9c51
# Full Hash: c11a6ced9c51a8c43592e0bc79a3de96c0b40efe
# Author: alwu <alwu@mozilla.com>
# Date: 2023-10-19 04:20:43
# Description:
#   Bug 1859600 - abort parsing NALU if the NALU size is incorrect. r=media-playback-reviewers,padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D191241
# ==============================================================================

diff -r d3d97163dde6 -r c11a6ced9c51 dom/media/test/crashtests/crashtests.list
--- a/dom/media/test/crashtests/crashtests.list	Wed Oct 18 17:15:25 2023 +0000
+++ b/dom/media/test/crashtests/crashtests.list	Wed Oct 18 17:15:25 2023 +0000
@@ -175,3 +175,4 @@
 load 1839193.html
 load 1850453.html
 test-pref(media.wmf.hevc.enabled,1) load 1859384.mp4
+test-pref(media.wmf.hevc.enabled,1) load 1859600.mp4
